project/Build.scala

```diff
@@ -928,6 +928,7 @@ object Build {
   // sbt plugin to use Dotty in your own build, see
   // https://github.com/lampepfl/dotty-example-project for usage.
   lazy val `sbt-dotty` = project.in(file("sbt-dotty")).
+    enablePlugins(ScriptedPlugin).
```

Comment:
Should we use the newer [SbtPlugin](https://developer.lightbend.com/blog/2018-07-02-sbt-1-2-0/#sbtplugin-for-plugin-development)?